src/go/cmd/http-relay-server/main.go

```diff
@@ -96,14 +98,15 @@ var (
 
 func main() {
 	flag.Parse()
-	log.SetFlags(log.LstdFlags | log.Lmicroseconds)
+	slog.SetDefault(slog.New(slog.NewJSONHandler(os.Stderr, nil)))
```

Comment:
Should this be the ilog JSON handler that uses `msg` instead of `message` (or vice versa, can't remember which one Cloud Logging likes).

Reply:
yes. thx